test/crawler/account/accounts_test.exs

```diff
@@ -0,0 +1,138 @@
+defmodule Crawler.Account.AccountsTest do
+  use Crawler.DataCase
```

Comment:
```suggestion
  use Crawler.DataCase, async: true
```

We can mark this test run as async by adding `async: true` @vnntsu, we had that on our Compass https://nimblehq.co/compass/development/code-conventions/elixir/ex-unit/#convention-3

You can read it more on  https://dockyard.com/blog/2019/02/13/understanding-test-concurrency-in-elixir

Reply:
@andyduong1920 Amazing. Now I know, thanks 🙏

Reply:
Fixed in [88f327a](https://github.com/vnntsu/elixir-ic/pull/31/commits/88f327a113b863f42b26afd460af432b4a1046ce)

Reply:
small issue in the commit message FYI @vnntsu ;) 
![image](https://user-images.githubusercontent.com/11751745/206083174-c1e28e7b-26a5-488a-8ca7-1c5c8487bc10.png)

Reply:
Good catch 🙏 I was focusing on this file `view_home_page_test.exs`, adding `false` but don't know what comment should be added 🤪